index.html

```diff
@@ -150,34 +150,30 @@ <h1>Text to Handwriting</h1>
             <div class="flex-1 page-container-super">
               <div>
                 <h2 style="margin-top: 0px;">Input</h2>
-                <label class="block" for="note">Type/Paste text here</label>
+                          <input type="file" name="inputfile"
+			  id="inputfile"> 
+		 <label class="block" for="note">Type/Paste text here</label>
               </div>
 
               <div class="flex-1 page-container">
                 <div class="page-a margined lines">
                   <div contenteditable="true" class="top-margin"></div>
                   <div class="display-flex left-margin-and-content">
                     <div contenteditable="true" class="left-margin"></div>
-                    <div class="paper-content" id="note" contenteditable="true">
-                      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-                      Ut rhoncus dui eget tortor feugiat iaculis. Morbi et dolor
-                      in felis viverra efficitur. Integer id laoreet arcu.
-                      Mauris turpis nibh, scelerisque sed tristique non,
-                      hendrerit in erat. Duis interdum nisl risus, ac ultrices
-                      ipsum auctor at. Aliquam faucibus iaculis metus sit amet
-                      tincidunt. Vestibulum cursus urna vel nunc imperdiet
-                      suscipit. Ut eu augue egestas, porta orci lobortis,
-                      pharetra orci. Morbi eu tellus quis nisi tempor accumsan
-                      nec eu urna. empor, ullamcorper tortor. Mauris quis
-                      fermentum augue. Phasellus ac nisl hendrerit, fringilla
-                      enim aliquam, placerat odio. Ut tortor nisi, pellentesque
-                      sed nisi at, hendrerit imperdiet turpis. Proin interdum
-                      porttitor metus at hendrerit. Proin ipsum nisl, lacinia
-                      vitae purus ullamcorper, maximus convallis sem. Sed nec
-                      lacinia lorem. Duis arcu metus, cursus non odio quis,
-                      tincidunt tempor augue. Donec eget vestibulum risus.
-                      Nullam blandit quis est vitae feugiat.
-                    </div>
+                    <div class="paper-content" id="note" contenteditable="true"  >
+			<pre id="output"></pre> 
+			<script type="text/javascript"> 
+			   document.getElementById('inputfile') 
+			   .addEventListener('change', function() { 
+			    var fr=new FileReader(); 
+			     fr.onload=function(){ 
+			    document.getElementById('output') 
+			   .textContent=fr.result; 
+			     } 
+			   fr.readAsText(this.files[0]); 
+			    }) 
+			   </script> 
```

Comment:
You can move this JavaScript to the JavaScript file and have the input HTML element as it was before. Then in the onload, you can replace the existing lorem ipsum text with the new text.